doc/nrf/ug_thingy53.rst

```diff
@@ -260,60 +187,158 @@ You can program the precompiled firmware image using one of the following ways:
+
+It is also necessary to add a flash controller overlay for the application core to allow data to be loaded from the external flash memory to the network core using the application core RAM.
+This is done because the network core cannot access the external flash memory directly.
+See :file:`thingy53_nrf5340_cpuapp.overlay` in a compatible sample for how this is done.
+
+.. _thingy53_app_mcuboot_bootloader:
 
 MCUboot bootloader
-  The sample must enable MCUboot bootloader and use the same configuration of the bootloader as all of the Thingy:53-compatible samples in the |NCS|.
-  This is needed to let MCUboot use the sample as an image update.
+==================
 
-  The MCUboot bootloader supports serial recovery and a custom command to erase settings storage partition.
-  Erasing settings parition is needed to ensure that an application would not be booted with incompatible content of settings partition.
+Each sample must enable the MCUboot bootloader and use the same configuration of the bootloader as all of the Nordic Thingy:53-compatible samples in the |NCS|.
+This ensures that the sample includes the MCUboot bootloader and that an MCUboot-compatible image is generated when the sample is built.
 
-In addition to the configurations mentioned above, an application compatible with Thingy:53 is expected to enable the following features:
+The MCUboot bootloader supports serial recovery and a custom command to erase the settings storage partition.
+Erasing the settings partition is needed to ensure that an application is not booted with incompatible content loaded from the settings partition.
 
-* USB CDC ACM as backend for logger.
-  The logs are provided using USB CDC ACM to allow accessing them without additional hardware.
+Use the following Kconfig options to enable the MCUboot bootloader:
 
-  Most of the Thingy:53-compatible applications and samples use only a single instance of USB CDC ACM that works as logger's backend.
-  No other USB classes are used.
-  These samples can share common USB product name, vendor ID, and product ID.
-  If a sample supports additional USB classes or more than one instance of USB CDC ACM, it must use dedicated product name, vendor ID, and product ID.
+* :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT`
+* :kconfig:option:`CONFIG_IMG_MANAGER`
+* :kconfig:option:`CONFIG_MCUBOOT_IMG_MANAGER`
+* :kconfig:option:`CONFIG_IMG_ERASE_PROGRESSIVELY`
 
-* DFU over-the-air through Simple Management Protocol over Bluetooth.
-  The application acts as GATT server and allows the connected Bluetooth Central to perform the firmware update for both application and network core.
-  The Bluetooth configuration is updated to allow quick DFU data transfer.
-  The application supports Simple Management Protocol (SMP) handlers related to:
+In addition, set the :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER` option to ``2`` and :kconfig:option:`CONFIG_MCUBOOT_IMAGE_VERSION` to ``"1.0.0+0"``.
```

Comment:
The image version could have any number I think. I would not refer to specific one here